.github/workflows/deploy.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will deploy to github pages after a pr is merged or commits are pushed to master
+
+name: Deploy to Github pages
+
+on:
+  pull_request:
+    branches: [master]
+  push:
+    branches: [master]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```

Comment:
Aren't we using `14.x`?

Reply:
I chose to keep the same version as the other action file. 

I am using `16,x` which is the latest LTS version.

https://github.com/ekumenlabs/telegram-tt/blob/3af7f1f89fb55f3689be01da33a822c47419b857/.github/workflows/tests.yml#L16

Reply:
Sounds good